source/migrating_from_legacy_products/index.html.md.erb

```diff
@@ -3,6 +3,121 @@ title: Migrate from our old frameworks
 weight: 210
 ---
 
-# Migrate from our old frameworks
+# Migrate to GOV.UK Frontend from our old frameworks
 
-Read about how to [update from GOV.UK Elements and Frontend Toolkit](https://test_moving_docs--govuk-design-system-preview.netlify.com/get-started/updating-your-code/).
+You should migrate to GOV.UK Frontend if you're using one of the following old ('legacy') frameworks:
+
+- GOV.UK Frontend Toolkit
+- GOV.UK Template
+- GOV.UK Elements
+
+We're no longer updating our old frameworks, and they do not meet the [Web Content Accessibility Guidelines (WCAG 2.1) accessibility standard](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps).
+
+After you include GOV.UK Frontend in your project, your service should look and behave the same as before. This means you can migrate one component at a time in your live service.
+
+Your service may be slightly slower while you migrate, because your users will need to download more CSS and JavaScript.
+
+## Audit your service's components
+
+To help you migrate, you should work with a designer to make a list of the components in your service, and compare it to [the components in GOV.UK Frontend](https://design-system.service.gov.uk/components/).
+
+You can use your list to see where you can replace similar elements in your service with one GOV.UK Frontend component. For example, if you have a range of similar buttons, you may be able to replace them all with the [button](https://design-system.service.gov.uk/components/button/) component.
+
+We recommend using a template engine like [Nunjucks](https://mozilla.github.io/nunjucks/) to abstract each component into a reusable macro (also called a 'partial' or 'template'). You can do this before or during your migration.
+
+If you use macros, your project will be easier to:
+
+- migrate, because your project will be organised the same way we organise the GOV.UK Design System
+- update when new versions of GOV.UK Frontend are released
+
+If you're using Node.js, you can use [our Nunjucks macros](https://design-system.service.gov.uk/styles/page-template/#nunjucks). You'll need to [install Nunjucks](https://mozilla.github.io/nunjucks/getting-started.html) first.
+
+## Include GOV.UK Frontend in your project
+
+1. [Install GOV.UK Frontend using node package manager (npm)](/installing_with_npm).
```

Comment:
I think Ollie and I spoke about this after it came up at the workshop, and we felt it would be better on the install page itself. But I forgot to add it! I'll take a look at this when I iterate the install page.

Reply:
Thanks @m-green 👍

Reply:
As discussed in https://github.com/alphagov/govuk-frontend/pull/1697#discussion_r399147027 we want to mention this here after all.